languages/fsharp/exercises/concept/floating-point-numbers/.meta/Example.fs

```diff
@@ -12,7 +12,10 @@ let private annualYield (balance: decimal): decimal =
 
 let annualBalanceUpdate (balance: decimal): decimal = balance + annualYield balance
 
-let amountToDonate (balance: decimal) (taxFreePercentage: double): int =
-    if balance > 0.0m
-    then int (balance * decimal (taxFreePercentage / 100.0 * 2.0))
-    else 0
+let amountToDonate (balance: decimal) (taxFreePercentage: float): int =
+    if balance > 0.0m then
+        int
+            (balance
+             * decimal (taxFreePercentage / 100.0 * 2.0))
+    else
+        0
```

Comment:
Is this spacing preferable? I personally find the original one more readable

Reply:
Good question! This is the formatting suggested by Fantomas, which is the F# source code formatting tool that I plan on using (similar like `dotnet format` for the C# track). I couldn't fully integrate Fantomas yet due to some open issues, but in the end I've managed to fix them both myself :D (see https://github.com/fsprojects/fantomas/pull/808 and https://github.com/fsprojects/fantomas/pull/809). I'll be working on integrating Fantomas soon-ish. In the mean time, I might be able to slightly rewrite the example to be more readable. Let me try and work on this!